src/sqlancer/postgres/PostgresErrors.java

```diff
@@ -0,0 +1,16 @@
+package sqlancer.postgres;
+
+import java.util.regex.Pattern;
+
+import sqlancer.common.query.ExpectedErrors;
+
+public final class PostgresErrors {
+
+    private PostgresErrors() {
+    }
+
+    public static void addExpressionErrors(ExpectedErrors errors) {
```

Comment:
This file is not necessary. You can consider adding the expected errors in `PostgresCommon.java` or the `PostgreCERTOracle.java`.

Reply:
Ok!

Reply:
As commented, can you remove this file?

Reply:
Done.